platform/ios/PolyPodApp/PolyPod/Authentication.swift

```diff
@@ -35,21 +35,31 @@ class Authentication {
         UserDefaults.standard.set(true, forKey: Authentication.disableCheckKey)
     }
     
-    func setUp(_ completeAction: @escaping (Bool) -> Void) {
-        authenticateLocally(reason: "auth_prompt_set_up") { success in
-            self.authenticated = success
-            UserDefaults.standard.set(true, forKey: Authentication.setUpKey)
+    
+    func setUp(newStatus: Bool, _ completeAction: @escaping (Bool) -> Void) {
```

Comment:
So, when user will disable the Authentication, they will be prompted with the same message as for when enabling it? Shouldn't there be another message when disabling the Auth?

Reply:
the prompt will depend on whether the setting is on or off - according to Lars's request above

Reply:
ok got it, maybe it would have been better(clearer) to have that reason based on the new status user wants to set, not on the old one. Anyway, what is wanted is achieved.